lib/galaxy/util/path/__init__.py

```diff
@@ -33,6 +33,17 @@
 from six.moves import map, zip
 log = logging.getLogger(__name__)
 
+def path_under_whitelist_dir(path, whitelist=None):
```

Comment:
I think this duplicates `safe_contains()` below? You should be able to use `safe_contains(configured import root directory, path, whitelist)` I think.